rc/qml/MediaViewer/PhotoEditorPage.qml

```diff
@@ -22,11 +22,11 @@ Page {
     id: page
     property string photo
     signal done(bool photoWasModified)
-
+    
     title: i18n.tr("Edit Photo")
-
+    
     head.backAction: Action {
-        iconName: "back"
+        iconName: "save"
```

Comment:
@TronFourtyTwo I wanted to know here if the back button also saves the changes :)